src/main/webapp/scripts/analyze.js

```diff
@@ -10,14 +10,10 @@ async function analyzeVideo() {
     }
   })
   .then(videoAnalysis => {
-    if (videoAnalysis.scoreAvailable) {
-      updateDom(videoAnalysis.score.toString(), 'sentiment-container');
-    } else {
-      updateDom('Video has no available captions or comments to analyze.',
-          'sentiment-container');
-    }
+    displaySentiment(videoAnalysis);
   })
   .catch(error => {
-    updateDom(error.message, 'sentiment-container');
+    displaySentiment({ scoreAvailable: false });
+    console.error(error);
```

Comment:
changed this to log the error in the console and then use the logic in displaySentiment to show an error to the user